packages/axes/src/compute.ts

```diff
@@ -120,30 +125,39 @@ export const getScaleTicks = (scale, spec) => {
     return scale.domain()
 }
 
-export const computeCartesianTicks = ({
+export const computeCartesianTicks = <Value extends string | number | Date>({
     axis,
     scale,
     ticksPosition,
     tickValues,
-    tickSize,
-    tickPadding,
-    tickRotation,
+    tickSize = NaN,
```

Comment:
Why do we need those? I'd suggest to add a comment if they're really required.

Reply:
I think this was to keep the types as optional, but I went ahead and removed it.